apps/trading/components/welcome-dialog/use-get-onboarding-step.ts

```diff
@@ -12,16 +12,20 @@ import { useGlobalStore } from '../../stores';
 const ONBOARDING_STORAGE_KEY = 'vega_onboarding';
 export const useOnboardingStore = create<{
   dialogOpen: boolean;
+  walletDialogOpen: boolean;
```

Comment:
(non blocking): This is not a good variable name as its not controlling a dialog. Just the state within the dialog. Lets not change it though to avoid conflicts with develop